src/Controller/ProjectController.php

```diff
@@ -60,6 +62,7 @@ public function show(Project $project): Response
 
     /**
      * @Route("/{id}/edit", name="project_edit", methods={"GET","POST"})
```

Comment:
You'll probably have a problem using a {id} parameter here without requirements.